pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -267,22 +268,35 @@ func (r *ReconcileDirectVolumeMigrationProgress) reportContainerStatus(pvProgres
 		if transferRate != "" {
 			pvProgress.Status.LastObservedTransferRate = transferRate
 		}
+		pvProgress.Status.Runtime = nil
 	case !containerStatus.Ready && containerStatus.LastTerminationState.Terminated != nil && containerStatus.LastTerminationState.Terminated.ExitCode != 0:
 		// pod has a failure, report last failure reason
 		pvProgress.Status.PodPhase = kapi.PodFailed
 		pvProgress.Status.LogMessage = containerStatus.LastTerminationState.Terminated.Message
+		exitCode := containerStatus.LastTerminationState.Terminated.ExitCode
+		pvProgress.Status.ExitCode = &exitCode
+		pvProgress.Status.Runtime = &metav1.Duration{Duration: containerStatus.LastTerminationState.Terminated.FinishedAt.Sub(containerStatus.LastTerminationState.Terminated.StartedAt.Time)}
```

Comment:
Prefer using `.Round(time.Second)` to round off duration to nearest second value to avoid fractional duration

Reply:
@pranavgaikwad I think the `metav1.Duration` will do the pretty formatting for us

Reply:
+1 `metav1.Duration` should handle this